lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -4243,11 +4257,11 @@ class JVPEmitter final
   }
 
   static SILBuilder
-  initializeDifferentialAndBuilder(ADContext &context, SILFunction *original,
-                                   SILDifferentiableAttr *attr,
-                                   LinearMapInfo *linearMapInfo) {
+  initializeDifferentialBuilder(ADContext &context, SILFunction *original,
```

Comment:
I'm very surprised this function existed. It's pretty specific to differentials and has only 2 lines. Inline this?

Reply:
Done in https://github.com/apple/swift/pull/27590/commits/b2481dee4ded16998f0d03dfdf216f66557813c1.